### DISCUSSION TITLE
Additional option to use...

### DISCUSSION THREAD

**Discussion by ASTIN-UA** [porrey/Virtual-ZPL-Printer#55] [Ideas]
Could you add a few options? Here is what I would like to see:

1.  Run the program at startup 
2.  Minimize to tray.  
3. Tray minimize at startup

And what about translation to other langs? Can translate to Ukrainian.))

Thank you in advance for your work!

**Comment by porrey** [comment]
I will consider these enhancements. I will do the work to support multiple languages if you are willing to do the translations and testing for your language. I am using some AI tools to initiate the translation but I suspect it will need some tweaking.

**Comment by ASTIN-UA** [comment]
Of course I will translate all needed texts. And I will test all of this without any problem. You don't need any AI to translate it for me. Just give me texts. )) Your programm is no so big for using AI to translate it))

**Comment by porrey** [comment]
I have used AI to get it started, mainly to ensure it works and that the UI adjusts accordingly. I have no way of knowing if it is accurate or not, so having people that natively speak the language provide the translations is the best option.

When the first version is ready for testing I will document the steps to do a pull request to update the language files.

Sneak Peek:

<img width="778" alt="Screenshot 2024-02-21 at 8 30 32 PM" src="https://github.com/porrey/Virtual-ZPL-Printer/assets/12475115/e157bcce-8b23-46c0-b350-c6432ec1d818">

<img width="782" alt="Screenshot 2024-02-21 at 8 31 11 PM" src="https://github.com/porrey/Virtual-ZPL-Printer/assets/12475115/81b2b522-3dcb-49eb-8c59-a5777c5c1813">

**Comment by ASTIN-UA** [comment]
Hi. It's not bad, as for AI. It's my correction for this screen:

![image](https://github.com/porrey/Virtual-ZPL-Printer/assets/17588757/58b037bc-56af-40d7-b173-e4eaa1e7559a)

I'm sory, but I can't help you with Spanish. It's not my native language. 
About this:
![image](https://github.com/porrey/Virtual-ZPL-Printer/assets/17588757/95266edc-b262-4769-95ee-847f8a25b479)

I prefer to use label as етикетка, and labels as етикеток. But both options can be considered correct.

"Idle" almost = stopped. So we can use "Зупинено". 
And, when printer is active, we can use "Слухаю". But it's word-for-word translation. More correct as for me will "Очікую". In English it's "Waiting".

So, we can continue to translate)

**Comment by porrey** [comment]
Thanks. Once I have complete all the conversions from fixed strings to resource strings, I will send you instructions to update the xml file and you can set the string to whatever you think best. I expect to be done in the next week or two.

**Comment by ASTIN-UA** [comment]
Super! Then I'll be waiting)

**Comment by porrey** [comment]
The installer is available under the branch **multi-language-support**. Please try this version and let me know if you have any issues. If not I will merge it to the main branch.
